src/main/java/com/datadog/api/v2/client/api/AuditApi.java

```diff
@@ -183,6 +187,70 @@ public CompletableFuture<AuditLogsEventsResponse> listAuditLogsAsync(
             });
   }
 
+  public PaginationIterable<AuditLogsEvent> listAuditLogsWithPagination() throws ApiException {
+    String resultsPath = "getData";
+    String valueGetterPath = "getMeta.getPage.getAfter";
+    String valueSetterPath = "pageCursor";
+    Boolean valueSetterParamOptional = true;
+    Integer limit;
+    ListAuditLogsOptionalParameters parameters = new ListAuditLogsOptionalParameters();
+
+    if (parameters.pageLimit == null) {
+      limit = 10;
+      parameters.pageLimit(limit);
+    } else {
+      limit = parameters.pageLimit;
+    }
+
+    LinkedHashMap<String, Object> args = new LinkedHashMap<String, Object>();
+    args.put("optionalParams", parameters);
+
+    PaginationIterable iterator =
+        new PaginationIterable(
+            this,
+            "listAuditLogs",
+            resultsPath,
+            valueGetterPath,
+            valueSetterPath,
+            valueSetterParamOptional,
+            limit,
+            args);
+
+    return iterator;
+  }
+
```

Comment:
Please add a docstring to the 2 new methods as well. We shouldn't duplicate docs, just add a link like the async version.